libraries/bot-azure/pom.xml

```diff
@@ -80,6 +80,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-storage-blob</artifactId>
+      <version>12.10.0</version>
+    </dependency>
```

Comment:
As soon as `AzureQueueStorage` is merged, we need to rebase this branch or the base one with `main` as this is already added there.